events/proto/transformed.proto

```diff
@@ -0,0 +1,8 @@
+
+syntax = "proto3";
+
+message Transformed {
+    Header header = 1;
+    MessageInfo incoming_message = 2;
+    repeated Created transformed_messages = 3;
```

Comment:
this feels heavy handed...and could be a really big array... but could also be nice to have rolled up in one place?